src/global.ts

```diff
@@ -98,6 +99,10 @@ export class Global {
    */
   public static async createDir(dirPath?: string): Promise<void> {
     dirPath = dirPath ? path.join(Global.SFDX_DIR, dirPath) : Global.SFDX_DIR;
-    await mkdirp(dirPath, '700');
+    try {
+      await fs.promises.mkdir(dirPath, { recursive: true, mode: 0o700 });
```

Comment:
this is worth a special UT test on windows (or manual check via repl) since the `mode` option isn't supported.  It's not clear from the node docs if that means "it doesn't do anything" or "it throws"

https://nodejs.org/api/fs.html#fsmkdirpath-options-callback

Reply:
`mod` only runs if platform is NOT win32